src/pymor/vectorarrays/block.py

```diff
@@ -178,8 +189,9 @@ def __eq__(self, other):
     def dim(self):
         return sum(subspace.dim for subspace in self.subspaces)
 
-    def zeros(self, count=1, reserve=0):
-        return BlockVectorArray([subspace.zeros(count=count, reserve=reserve) for subspace in self.subspaces], self)
+    def zeros(self, count=1, reserve=0, dtype=None):
+        return BlockVectorArray([subspace.zeros(count=count, reserve=reserve, dtype=None)
+                                 for subspace in self.subspaces], self)
```

Comment:
If I remember our discussion correctly, `zeros` should probably have no `dtype` argument and there should be no `dtype=None` in the `subspace.zeros` call?